pkg/generators/golang/builders_generator.go

```diff
@@ -556,6 +556,8 @@ func (g *BuildersGenerator) fieldType(attribute *concepts.Attribute) *TypeRefere
 	typ := attribute.Type()
 	var ref *TypeReference
 	switch {
+	case typ.IsInterface():
+		ref = g.types.ValueReference(typ)
```

Comment:
Not sure I understand why.
Earlier you added `InterfaceType` to `IsScalar()` func

Reply:
This is actually a hack. `interface{}` is nullable, of course, but it's already a pointer. Letting it run as `NullableReference` means that the generator marks it as `*interface{}`, which is not correct - forcing it to `ValueReference` renders it without the `*`. To be more accurate I can change `NullableReference` to skip interface types (I'll try that and report back)

Reply:
Alright, I've made the code cleaner. It removes a few explicit `IsInterface()` checks and relies on `IsScalar()` more instead. Then the `NullableReference` just checks whether `IsScalar && !IsInterface` to prefix the `*`